jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/helper/DockerFileUtil.java

```diff
@@ -42,282 +42,288 @@
 
 /**
  * Utility class for dealing with dockerfiles
+ * 
  * @author roland
  * @since 21/01/16
  */
 public class DockerFileUtil {
-    private static final String ARG_PATTERN_REGEX = "\\$(?:\\{(.*)\\}|(.*))";
```

Comment:
This class was completely reformatted and is really hard to track the changes (at least in the GH interface).
Could you please revert the reformatting so that it's clearer for us to see what was changed here?